From dd147ab110e71ea0f1091923c9230ade01f226d4 Mon Sep 17 00:00:00 2001
From: Victor Zhestkov <Victor.Zhestkov@suse.com>
Date: Fri, 28 Oct 2022 13:19:23 +0300
Subject: [PATCH] Detect module.run syntax

* Detect module run syntax version

* Update module.run docs and add changelog

* Add test for module.run without any args

Co-authored-by: Daniel A. Wozniak <dwozniak@saltstack.com>
---
 changelog/58763.fixed | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 changelog/58763.fixed

diff --git a/changelog/58763.fixed b/changelog/58763.fixed
new file mode 100644
index 0000000000..53ee8304c0
--- /dev/null
+++ b/changelog/58763.fixed
@@ -0,0 +1 @@
+Detect new and legacy styles of calling module.run and support them both.
-- 
2.37.3


